angular/projects/researchdatabox/form/src/app/component/repeatable.component.ts

```diff
@@ -127,37 +127,41 @@ export class RepeatableComponent extends FormFieldBaseComponent<Array<unknown>>
     // Create new form field.
     const model = this.formService.createFormFieldModelInstance(elemEntry, this.newElementFormConfig?.validatorDefinitions);
 
-    // Set the value if provided.
-    if (model && value) {
-      model.setValue(value);
-    }
+    elemEntry.model = model;
```

Comment:
The service method `this.formService.createFormFieldModelInstance` already sets the model to the `FormFieldCompMapEntry` var. In fact, the method returns `elemEntry.model`, so this line sets the model to the same thing it was already set to.